src/app/core/data/root-data.service.ts

```diff
@@ -106,5 +124,12 @@ export class RootDataService {
   findAllByHref(href: string | Observable<string>, findListOptions: FindListOptions = {}, useCachedVersionIfAvailable = true, reRequestOnStale = true, ...linksToFollow: FollowLinkConfig<Root>[]): Observable<RemoteData<PaginatedList<Root>>> {
     return this.dataService.findAllByHref(href, findListOptions, useCachedVersionIfAvailable, reRequestOnStale, ...linksToFollow);
   }
+
+  /**
+   * Set to sale the root endpoint cache hit
+   */
+  invalidateRootCache() {
+    this.requestService.setStaleByHrefSubstring('server/api');
```

Comment:
I'm not sure we can assume that the root REST API path will always include "server/api".  What if someone deploys the REST API webapp at the root path?  Then the REST API would be under "/api" instead of "/server/api".   In other words, while our REST API is hardcoded to the "/api" endpoint, what comes before that path is completely configurable/customizable... it could be setup as `/server/api` (the default), or `/dspace/api`, or just `/api`.

So, I think may need to read the REST API root path from `config.*.yml`, or we may need to assume the root path just includes "/api"

Reply:
fxed using the getRootHref method instead of harcoding the url